reader.py

```
@@ -170,59 +206,23 @@ def keyPressEvent(self, event):
+                self.menu_bar_widget.show()
+            self.hide_menu_timer.stop()
+        
+        else:
+            if self.menu_bar_widget.isVisible() and not self.hide_menu_timer.isActive():
+                self.hide_menu_timer.start(1500)
+        super().mouseMoveEvent(event)
 
 if __name__ == "__main__":
     app = QApplication(sys.argv)
-    reader = Comic(r"D://comic_library//Juggernaut - No Stopping Now TPB (March 2021).cbz")
+    reader = Comic(r"D://comic_library//comic_examples//Juggernaut - No Stopping Now TPB (March 2021).cbz")
```

Pylint (3.3.7)

[convention] Line too long (107/100)

(C0301)